src/TypeScriptGenerator.ts

```diff
@@ -51,14 +51,14 @@ export const generate: TypeGenerator["generate"] = (schema, node, options) => {
   const printer = ts.createPrinter({ newLine: ts.NewLineKind.LineFeed });
 
   const resultFile = ts.createSourceFile(
-    "graphql-def.ts",
+    "grapghql-def.ts",
```

Comment:
Os this a typo?

Reply:
IT was a typo previously, I just reverted the whole commit.

Reply:
```suggestion
    "graphql-def.ts",
```